backend/src/api.ts

```diff
@@ -105,41 +105,32 @@ router.post('/logout', (req: Request, res: Response) => {
 router.get('/allRoles', getAllRoles);
 
 // Members
-router.get('/allMembers', async (_, res: Response) => {
+router.get('/allMembers', async (_, res) => {
   const handled = await allMembers();
   res.status(handled.status).json(handled);
 });
-router.get('/getMember/:email', async (req: Request, res: Response) => {
+router.get('/getMember/:email', async (req, res) => {
   const handled = await getMember(req, res);
   res.status(handled!.status).json(handled);
 });
-router.post('/setMember', async (req: Request, res: Response) => {
+router.post('/setMember', async (req, res) => {
   const handled = await setMember(req, res);
   res.status(handled!.status).json(handled);
 });
-router.delete('/deleteMember/:email', async (req: Request, res: Response) => {
+router.delete('/deleteMember', async (req, res) => {
```

Comment:
mistake in merge?

Reply:
I think I pulled the master branch changes into this branch once Henry fixed the delete member and edit teams problems? I didn't make these changes not sure what happened

Reply:
I think it's better to remove all the changes in the backend in this PR, since they are almost all unintended.

Reply:
just went back and fixed all the changes, it should be good now